app/src/main/java/org/simple/clinic/teleconsultlog/prescription/medicines/TeleconsultMedicinesView.kt

```diff
@@ -74,6 +75,9 @@ class TeleconsultMedicinesView(
   @Inject
   lateinit var screenKeyProvider: ScreenKeyProvider
 
+  @Inject
+  lateinit var drugFrequencyToLabelMap: Map<DrugFrequency?, DrugFrequencyLabel>
```

Comment:
Yes, as the label we display for `null` drug frequencies is `None`

Reply:
It is intentional for now. Ideally, I would have preferred to have a specific type for handling `null`, but that change would impact a lot of other usages. So for now we are using the `null` as a key.

Reply:
Understood. We should correct this though.

Reply:
It is a little bit tricky since the `DrugFrequency` is a payload object. So we cannot add a `None` type there unless the server also handles that type (or we map that to nullable in the `DrugPayload` object). So I am lost on how to properly handle this scenario 🤔